zenoh/src/lib.rs

```diff
@@ -287,6 +290,7 @@ where
 
     /// Restrict the matching queries that will be receive by the [`Queryables`](crate::queryable::Queryable)
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should not be configurable at session level but only per `declare_subscriber` and `declare_queryable`-basis.